test/functions/whereClauses/whereClauseParamArgMismatch3.good

```diff
@@ -1,2 +1,2 @@
 whereClauseParamArgMismatch3.chpl:10: error: unresolved call 'foo(1.2)'
-whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(8))
+whereClauseParamArgMismatch3.chpl:1: note: candidates are: foo(param x: uint(64))
```

Comment:
I think we could also remove this test.  I think when I was writing these, I was under the impression that non-default integer sizes caused distinct problems or went down distinct code paths, but I've since become convinced that I was high at the time.  I can clean this up after the merge if you don't want to mess further with it.